tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3830,6 +3830,37 @@ public function theResponseShouldContain(string $method, string $user, string $s
 		}
 	}
 
+	/**
+	 * @Then as user :user the key :key from PROPFIND response should match with sharedWithMe driveItemId of share :resource
+	 *
+	 * @param string $user
+	 * @param string $key
+	 * @param string $resource
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function asUserTheKeyFromPropfindResponseShouldMatchWithSharedwithmeDriveitemidOfShare(string $user, string $key, string $resource): void {
+		$xmlRes = $this->featureContext->getResponseXml();
```

Comment:
```suggestion
		$xmlRes = $this->featureContext->getResponseXml();
```
name should not short